preload.js

```diff
@@ -70,6 +72,15 @@
   window.libphonenumber.PhoneNumberFormat = require('google-libphonenumber').PhoneNumberFormat;
   window.nodeNotifier = require('node-notifier');
 
+  const { autoOrientImage } = require('./js/modules/auto_orient_image');
+  window.autoOrientImage = autoOrientImage;
+
+  // ES2015+ modules
+  window.Whisper = window.Whisper || {};
+  window.Whisper.Types = window.Whisper.Types || {};
```

Comment:
Created new `Whisper.Types` namespace to prevent collision for `Whisper.Message`. However, I am wondering if it’d make sense to introduce `window.Signal` for new modules. This would help us make a better distinction between old- and new-style modules.